sdk/java-sdk-spring/src/main/java/kalix/javasdk/annotations/Subscribe.java

```diff
@@ -152,6 +152,13 @@
      */
     String service();
 
+    /**
+     * In case you need to consume the same stream multiple times, each subscription should have a unique consumer group.
+     * <p>
+     * Changing the consumer group will lead to starting over from the beginning of the stream.
+     */
+    String consumerGroup() default "";
```

Comment:
I thought about a different name, but my conclusion was that it would be confusing to have the same semantics as Topic.consumerGroup, under another name.